app/src/main/res/layout/fragment_profile.xml

```diff
@@ -2,124 +2,262 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="@color/gray5"
+    android:orientation="vertical">
 
-    <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/app_bar"
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/scroll"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:clipToPadding="false"
+        android:paddingBottom="@dimen/nested_scroll_bottom_padding"
+        app:layout_behavior="com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior">
 
-        <com.google.android.material.appbar.CollapsingToolbarLayout
-            android:id="@+id/collapsing_toolbar_layout"
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/top_profile"
             android:layout_width="match_parent"
-            android:layout_height="240dp"
-            app:collapsedTitleTextAppearance="@style/AppTheme.Toolbar.Title"
-            app:contentScrim="?attr/colorPrimary"
-            app:expandedTitleGravity="top"
-            app:expandedTitleTextAppearance="@style/TransparentText"
-            app:layout_scrollFlags="scroll|exitUntilCollapsed|snap">
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <ImageView
```

Comment:
Use the already created layout that have the image and the color with transparency